QgisModelBaker/libqgsprojectgen/dbconnector/db_connector.py

```diff
@@ -172,6 +173,40 @@ def get_bags_of_info(self):
         '''
         return []
 
+    def get_omitted_layers(self):
```

Comment:
```suggestion
    def get_ignored_layers(self):
```

Can we agree on one taxonomy?

Reply:
I can do that.